src/kaleidoscope/plugin/LEDEffect-Chase.cpp

```diff
@@ -30,15 +30,15 @@ void LEDChaseEffect::TransientLEDMode::update(void) {
 
   // The red LED is at `pos_`; the blue one follows behind. `direction_` is
   // either +1 or -1; `distance_` is the gap between them.
-  int8_t pos2 = pos_ - (direction_ * parent_->distance_);
+  uint8_t pos2 = pos_ - (direction_ * parent_->distance_);
```

Comment:
(Commenting here because it's -close- to the issue.)

We sitll have a single use of LED_COUNT inside this plugin. It should die as part of this cleanup. (And then LED_COUNT should get a deprecation warning)